storage/series/reader.go

```diff
@@ -171,13 +175,15 @@ func (r Reader) readersWithBlocksMapAndBuffer(
 func (r Reader) FetchBlocks(
 	ctx context.Context,
 	starts []time.Time,
+	blockDuration time.Duration,
```

Comment:
nit: We use `blockSize` as the convention for passing the length of the block time window.